sources/Interop/WinTrust/um/wintrust/CAT_MEMBERINFO2.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/WinTrust.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public partial struct CAT_MEMBERINFO2
+    {
+        [NativeTypeName("GUID")]
+        public _GUID SubjectGuid;
```

Comment:
`_GUID` should be remapped to `Guid` (which will automatically add the appropriate `using System;`, etc)

Reply:
Fixed.